test/test_integration_credentials_keyring.py

```diff
@@ -50,38 +50,42 @@ def validate_credentials(self, machine, login, password):
         assert creds.login == login
         assert creds.password == password
 
-
     # *************** LOGIN CREDENTIALS TESTS ***************
 
     '''
     Validates that if a user configures the CLI in insecure mode and runs the command not in 
     insecure mode, then they will fail
     '''
+
     @integration_test(True)
     def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error(self):
-        shutil.copy(self.environment.path_provider.test_insecure_conjurrc_file_path, self.environment.path_provider.conjurrc_path)
+        shutil.copy(self.environment.path_provider.test_insecure_conjurrc_file_path,
+                    self.environment.path_provider.conjurrc_path)
         output = self.invoke_cli(self.cli_auth_params,
-                            ['login', '-i', 'admin', '-p', self.client_params.env_api_key], exit_code=1)
+                                 ['login', '-i', 'admin', '-p', self.client_params.env_api_key], exit_code=1)
         self.assertIn("The client was initialized without", output)
 
     '''
     Validates that if a user configures the CLI in insecure mode and runs a command in 
     insecure mode, then they will succeed
     '''
+
     @integration_test(True)
     @patch('builtins.input', return_value='yes')
     def test_cli_configured_in_insecure_mode_and_run_in_insecure_mode_passes(self, mock_input):
         self.invoke_cli(self.cli_auth_params,
-                        ['--insecure', 'init', '--url', self.client_params.hostname, '--account', self.client_params.account])
+                        ['--insecure', 'init', '--url', self.client_params.hostname, '--account',
+                         self.client_params.account])
 
         output = self.invoke_cli(self.cli_auth_params,
-                            ['--insecure', 'login', '-i', 'admin', '-p', self.client_params.env_api_key])
+                                 ['--insecure', 'login', '-i', 'admin', '-p', self.client_params.env_api_key])
         self.assertIn('Successfully logged in to Conjur', output)
 
     '''
     Validates a user can log in with a password, instead of their API key
     To do this, we perform the following:
     '''
+
     @integration_test()
     @patch('builtins.input', return_value='yes')
     def test_https_credentials_created_with_all_parameters_given(self, mock_input):
```

Comment:
I know I wrote this test and named it but this isn't really representative of what the test is checking (see the comment above) 
I think we should add a test that just runs and validate it is in the netrc explicitly 
```
    @integration_test()
    @patch('builtins.input', return_value='yes')
    def test_https_credentials_created_with_all_parameters_given(self, mock_input):
        self.invoke_cli(self.cli_auth_params,
                        ['login', '-i', 'admin', '-p', self.client_params.env_api_key])
        utils.get_credentials()
        self.validate_credentials(f"{self.client_params.hostname}", "someuser", extract_api_key_from_message)
```
and have another test with the content of this test
def test_https_credentials_user_can_login_successfully_when_another_user_is_already_logged_in(self, mock_input):

Reply:
this test should be for both keyring and netrc :)